chore: update HMR announce (#1457)
ngxs/store: publication/2019-10-21_announcing_ngxs_3_6/article.md

### HMR Plugin

- Feature: HMR Plugin - Add hmrIsReloaded utility [#1435](https://github.com/ngxs/store/pull/1435)

> If you make any changes in your application during the `ngOnDestroy` Angular lifecycle hook on any of your primary application components then it those changes may have any side effects for Hot Module Replacement.

Reviewer: ```suggestion
If you make any changes in your application during the `ngOnDestroy` Angular lifecycle hook on any of your primary application components then those changes could possibly have side effects for Hot Module Replacement.
```
